arviz/plots/compareplot.py

```diff
@@ -91,7 +91,7 @@ def plot_compare(
 
     yticks_labels = [""] * len(yticks_pos)
 
-    _information_criterion = ["waic", "loo"]
+    _information_criterion = ["waic", "loo", "WAIC", "LOO"]
     for information_criterion in _information_criterion:
         if information_criterion in comp_df.columns:

```

Comment:
I think we should change this this to be `comp_df.colums.str.lower()`

Reply:
Yes, but then the indexing below needs to change as well - or you generate a copy of comp_df and overwrite the columns.

Reply:
What do you think about overwriting comp_df columns in place?

Reply:
That would be a pretty big side effect...